storagedriver/azure/azure.go

```diff
@@ -174,11 +193,17 @@ func (d *Driver) WriteStream(path string, offset, size int64, reader io.ReadClos
 		}
 	}
 
-	if offset != resumableOffset {
-		return storagedriver.InvalidOffsetError{Path: path, Offset: offset}
+	if offset < resumableOffset {
```

Comment:
I guess there isn't a test currently enforcing this, but we must allow the user of the storagedriver to overwrite parts of a file, for instance writing chunks from the same offset twice in a row.